python/cvmfs/repository.py

```diff
@@ -292,10 +459,9 @@ def retrieve_certificate(self):
         return Certificate(certificate)
 
 
-    @abc.abstractmethod
     def retrieve_file(self, file_name):
-        """ Abstract method to retrieve a file from the repository """
-        pass
+        """ Method to retrieve a file from the repository """
```

Comment:
This method can go completely, I think. Shouldn't it be replaced by a call to the `Fetcher`?